tests/ansible_lint_rules/NoFormattingInWhenRule.py

```diff
@@ -0,0 +1,32 @@
+from ansiblelint import AnsibleLintRule
```

Comment:
https://github.com/willthames/ansible-lint/blob/master/README.md I just read the docs (and then found out they were wrong, and fixed them by reading the code). This file also lives on, in an updated and improved form, in my PR against upstream. https://github.com/willthames/ansible-lint/pull/262

This file should not last long, if mine gets merged up there.